csbuild/_testing/functional_test.py

```diff
@@ -254,7 +254,7 @@ def _handleStderr(shared, msg):
 		commandThread.join()
 
 		ansiEscape = re.compile(r'\x1b[^m]*m')
-		returncode, output, errors = _shared.ret
+		returncode, output, errors = _shared.ret # pylint: disable=unpacking-non-sequence
```

Comment:
I wonder why pylint thinks this is a non-sequence... can we better type-hint `_shared.ret` to get pylint happy instead of disabling the check?